### PR TITLE
docs(mux-uploader): Update README to replace copy pasta from mux-player.

### PR DESCRIPTION
**Pull request by clearlyTHUYDOAN**
## Description

Some text from copying over `mux-player`'s markdown table was leftover 🍝 

Updated text to be relevant to `mux-uploader` 🌈

### CODE REVIEW

File: packages/mux-uploader/README.md [hunk 1]
```diff
@@ -61,10 +61,7 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
 <body>
   <p></p>
 
-  <mux-uploader
-    url="authenticated-url"
-    id="uploader"
-  ></mux-player>
+  <mux-uploader url="authenticated-url" id="uploader"></mux-uploader>
 </body>
```
Comment (dylanjha) [inline packages/mux-uploader/README.md:65]: can we expand on this example, or add another example about how to use `<mux-uploader-drop>`? Based on these docs I'm not sure how to use that other component

File: packages/mux-uploader/README.md [hunk 2]
```diff
@@ -77,9 +74,9 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
 | `url`                | `string`  | The authenticated URL that your file will be uploaded to. Check out the [direct uploads docs](https://docs.mux.com/guides/video/upload-files-directly#1-create-an-authenticated-mux-url) for how to create one. Required. | `undefined` |
 | `id`                 | `string`  | An ID that allows `mux-uploader-drop` to locate `mux-uploader`. Required.                                                                                                                                                 | N/A         |
```
Comment (dylanjha) [inline packages/mux-uploader/README.md:75]: is `id` *always* required? even if I'm not using `mux-uploader-drop`?

File: packages/mux-uploader/README.md [hunk 3]
```diff
@@ -77,9 +74,9 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
...
-| `status`             | `boolean` | This is an arbitrary title for your video that will be passed in as metadata into Mux Data. Adding a title will give you useful context in your Mux Data dashboard. (optional, but encouraged)                            | false       |
+| `upload-in-progress` | `boolean` | Toggles visual status of progress bar while upload is n progress.                                                                                                                                                         | false       |
+| `upload-error`       | `boolean` | Toggles visual status of progress bar when upload encounters an error.                                                                                                                                                    | false       |
+| `status`             | `boolean` | Toggles text status of progress bar, which is a percentage by default.                                                                                                                                                    | false       |
```
Comment (dylanjha) [inline packages/mux-uploader/README.md:79]: looks better 👍🏼 

Can you clarify how someone would use these attributes? As a developer what is the use case for when I would want to show or hide the progress bar or error?

Or are these attributes that get sent internally by the web component and I'm not supposed to be manipulating them from the outside?

I'm not quite following `status` which says is `boolean` but then says it is a percentage by default. What is the alternative to percentage?

**Review by dylanjha — CHANGES_REQUESTED** [review]

File: packages/mux-uploader/README.md (hunk 3 above)
Comment (clearlyTHUYDOAN) [reply packages/mux-uploader/README.md:79]: Will work on clarifying! Thanks for providing insight as a person totally new to the docs.

**Comment by clearlyTHUYDOAN** [comment]
@dylanjha Added some more examples, clarified certain attributes, found some non-blocking TO-DO items, and renamed a custom event to be more relevant to its use cases i.e. `mux-drop` => `file-ready` (also used for click events).

**Review by dylanjha — APPROVED** [review]
Love it, much better 👏🏼 

I may have some suggestions for future iterations, but for now this is a big improvement!

**Pull request merged**